hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -720,6 +720,14 @@ public void renameKey(String volumeName, String bucketName,
         keyPrefix, maxKeys);
   }
 
+  @Override
+  public boolean recoverTrash(String volumeName, String bucketName,
+      String keyName, String destinationName) throws IOException {
```

Comment:
NIT: can we rename destinationName  to destinationBucket to be consistent?

Reply:
Thanks @xiaoyuyao for this NIT.
Fixed.